<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 	rgb(245,255,250);
background-color: 	rgb(238,238,228);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoginpageButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Calibri (Body)&quot;;
background-color: rgb(60,179,113);
color: rgb(255, 255, 255);
border-radius:10px;



</string>
    </property>
    <property name="text">
     <string>back to login page</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Calibri (Body)&quot;;
background-color: rgb(21,76,121);
color: rgb(255, 255, 255);
border-radius:5px;



</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Calibri (Body)&quot;;
color:   rgb(21,76,121);
 </string>
    </property>
    <property name="text">
     <string>please enter your username that we send you new password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecoveryButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 8pt &quot;Calibri (Body)&quot;;
background-color: rgb(60,179,113);
color: rgb(255, 255, 255);
border-radius:10px;



</string>
    </property>
    <property name="text">
     <string>recovery</string>
    </property>
   </widget>
   <zorder>username</zorder>
   <zorder>label</zorder>
   <zorder>RecoveryButton</zorder>
   <zorder>LoginpageButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
